chip.wdl

```diff
@@ -196,6 +198,8 @@ workflow chip {
         Int xcor_time_hr = 24
         String xcor_disks = 'local-disk 100 HDD'
 
+        Int subsample_ctl_mem_mb = 16000
```

Comment:
Why not use GB? Makes it more legible:
```suggestion
        Int subsample_ctl_mem_gb = 16
```

Reply:
Other resource parameters are still at `_mb` level. I was actually planning to change all of them to `_gb` for next refactoring.

Reply:
Sounds good.